packages/medusa/src/services/claim.js

```diff
@@ -430,21 +456,38 @@ class ClaimService extends BaseService {
         .withTransaction(manager)
         .emit(ClaimService.Events.REFUND_PROCESSED, {
           id,
+          no_notification: result.no_notification,
         })
 
       return result
     })
   }
 
-  async createShipment(id, fulfillmentId, trackingLinks, metadata = []) {
+  async createShipment(
+    id,
+    fulfillmentId,
+    trackingLinks,
+    config = {
+      metadata: {},
+      no_notification: undefined,
+    }
+  ) {
+    const { metadata, no_notification } = config
+
     return this.atomicPhase_(async manager => {
       const claim = await this.retrieve(id, {
         relations: ["additional_items"],
       })
 
+      const evaluatedNoNotification =
+        no_notification !== undefined ? no_notification : claim.no_notification
+
       const shipment = await this.fulfillmentService_
         .withTransaction(manager)
-        .createShipment(fulfillmentId, trackingLinks, metadata)
+        .createShipment(fulfillmentId, trackingLinks, {
+          metadata,
+          no_notification: evaluatedNoNotification,
```

Comment:
Again - method signature in `FulfillmentService` is not in line with this call. Update `/src/services/fulfillment.js`